### PR TITLE
chore: update release info to v5.30

### PR DESCRIPTION
**Pull request by barmac**
### Proposed Changes

test

Related to #4691

### Checklist

To ensure you provided everything we need to look at your PR:

* [ ] __Brief textual description__ of the changes present
* [x] __Visual demo__ attached
* [ ] __Steps to try out__ present, i.e. [using the `@bpmn-io/sr` tool](https://github.com/bpmn-io/sr)
* [x] Related issue linked via `Closes {LINK_TO_ISSUE}` or `Related to {LINK_TO_ISSUE}`

### CODE REVIEW

**Comment by nikku** [comment]
My questions:

* Should we highlight the search improvements?
* Should we highlight the core modeling (keyboard related) changes?

**Comment by barmac** [comment]
I think search makes sense, but I couldn't find any good message for keyboard binding. I know that it's useful for the developers, but what is the impact on the user? Do you have any suggestion?

**Comment by barmac** [comment]
> > I couldn't find any good message for keyboard binding.
> 
> Agreed that this is an implementation detail. We definitely want to give folks a heads up internally though (and we do also externally via the CHANGELOG).

Yes, I will talk about this in the release presentation for sure.

**Comment by barmac** [comment]
I updated the release info so that search is included.

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,12 +44,16 @@ export function ReleaseInfo(props) {
...
         </li>
         <li>
-          <h4>Support for new Camunda versions</h4>
-          Model diagrams for upcoming 8.7 and 7.23 versions.
+          <h4>Improved variables panel</h4>
+          Toggle the variables panel with a shortcut or from the application footer.
+        </li>
+        <li>
+          <h4>More accurate search</h4>
+          Elements search now displays the exact matches first. This allows to find elements faster.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:56]:
```suggestion
          Find more things inside a BPMN diagram, using improved search and filter functionality.
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 2]
```diff
@@ -44,12 +44,16 @@ export function ReleaseInfo(props) {
...
+          Define user task listeners in your Camunda 8.7 diagrams, and get guidance along way.
         </li>
         <li>
-          <h4>Support for new Camunda versions</h4>
-          Model diagrams for upcoming 8.7 and 7.23 versions.
+          <h4>Improved variables panel</h4>
+          Toggle the variables panel with a shortcut or from the application footer.
+        </li>
+        <li>
+          <h4>More accurate search</h4>
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:55]:
```suggestion
          <h4>Improved on canvas search</h4>
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 3]
```diff
@@ -44,12 +44,16 @@ export function ReleaseInfo(props) {
...
-          <h4>New form components</h4>
-          Attach files to your Camunda 8.7 forms and get guidance as you upgrade.
+          <h4>Support for task listeners in Camunda 8</h4>
+          Define user task listeners in your Camunda 8.7 diagrams, and get guidance along way.
         </li>
         <li>
-          <h4>Support for new Camunda versions</h4>
-          Model diagrams for upcoming 8.7 and 7.23 versions.
+          <h4>Improved variables panel</h4>
+          Toggle the variables panel with a shortcut or from the application footer.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:52]:
```suggestion
          Access the variables panel easily from the application footer or toggle it through a keyboard shortcut.
```

**Comment by nikku** [comment]
I'd frame the "more acurate search" as "improved search", because it was already "acurate" before, we wanted results to be more "relevant" :)

**Comment by nikku** [comment]
Of course [q] is if we want to feature the ["update available indicator"](https://github.com/camunda/camunda-modeler/issues/4407). From what I understand you mentioned you don't want to feature it @barmac? Will be part of changelog anyway.

**Comment by barmac** [comment]
> Of course [q] is if we want to feature the ["update available indicator"](https://github.com/camunda/camunda-modeler/issues/4407). From what I understand you mentioned you don't want to feature it @barmac? Will be part of changelog anyway.

The thing is if you update on time, you won't see the new button. There's also no way to demo it.

**Comment by barmac** [comment]
I've applied your suggestions, and will update the screenshot. Thank you!

**Comment by barmac** [comment]
I am merging this for the RC, but we can still comment and adjust
@lmbateman @crobbins215

**Comment by crobbins215** [comment]
Suggestions, some just minor, main one I feel more strongly about is canvas search. I'm not keen on "Find more things"

Support for task listeners in Camunda 8
Define user task listeners in your Camunda 8.7 diagrams, with helpful guidance provided every step of the way.

Enhanced variables panel
Quickly access the variables panel from the application footer or toggle it using a keyboard shortcut.

Improved on canvas search
Locate elements and details more efficiently within BPMN diagrams, thanks to upgraded search and filtering capabilities.

Bug fixes and enhancements
This release includes bug fixes and additional minor improvements to enhance your experience.

**Comment by barmac** [comment]
Let me have a screenshot with your suggestions

**Comment by barmac** [comment]
@crobbins215 suggestions:

_Users_maciej_workspace_bpmn-io_camunda-modeler_client_build_index html

**Comment by barmac** [comment]
I opened a PR for update: https://github.com/camunda/camunda-modeler/pull/4716

**Pull request merged**